### PR TITLE
LPD-25679 Add onSetData prop to RichTextLocalized component

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: https://liferay.atlassian.net/browse/LPD-25679

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:sf

**Review by igor-franca — CHANGES_REQUESTED** [review]
Hey @danvb112 , could you sort these test alphabetically.

I undestand that the last unit test fit better in the end. Since it's work as a complex case involving all the other cases. 

But related with the tests that not have logic order, we can sort them. 

Nice work with these tests!

**Comment by danvb112** [comment]
ci:test:sf

File: modules/test/playwright/pages/notification-web/NotificationTemplatesPage.ts [hunk 1]
```diff
@@ -19,9 +19,11 @@ export class NotificationTemplatesPage {
...
 			.getByRole('menuitem')
 			.filter({hasText: 'Email'});
-		this.frontEndDatasetItemActions = page.getByRole('button', {
-			name: 'Actions',
-		});
+		this.frontEndDatasetItemActions = page
+			.getByRole('button', {
+				name: 'Actions',
+			})
+			.nth(-1);
```
[the remark this answers is not in the document]
Comment (danvb112) [reply modules/test/playwright/pages/notification-web/NotificationTemplatesPage.ts:26]: Hey @igor-franca, I think we can extract this to a function only if necessary, all the cases we have now exclude the last item, if for some reason we need to delete in some specific order, we can extract and pass the nth as a parameter

[the remark this answers is not in the document]
Comment (igor-franca) [reply modules/test/playwright/pages/notification-web/NotificationTemplatesPage.ts:26]: Ok, but keep in mind that we can perform other actions besides deleting the entity from the FDS. But yes, we can improve by demand 😄

**Comment by danvb112** [comment]
ci:test:relevant

**Comment by igor-franca** [comment]
ci:test:playwright

**Comment by danvb112** [comment]
CI failures are not related! Sending directly to Brian

**Comment by danvb112** [comment]
Sent to: https://github.com/brianchandotcom/liferay-portal/pull/150421

**Pull request closed without merging**